mobius-android/src/main/java/com/spotify/mobius/android/LiveQueue.java

```diff
@@ -50,7 +50,7 @@
    * substitutes null for the optional observer. See linked method doc for full info.
    */
   void setObserver(
-      @Nonnull LifecycleOwner lifecycleOwner, @Nonnull Observer<? super T> liveEffectsObserver);
+      @Nonnull LifecycleOwner lifecycleOwner, @Nonnull Observer<T> liveEffectsObserver);
```

Comment:
I think we should keep this `? super T`.

Reply:
(the rationale being that it improves the flexibility of the API; you can use observers of super-types of T that way)

Reply:
I know this function wasn't affected by the same type issues as the one below, but I thought it would be nicer to keep them both similar in what they expect, so that they can be at least somewhat re-used with the same/similar function (wrapped in different observers)